hail/python/hail/backend/service_backend.py

```diff
@@ -366,6 +366,7 @@ async def _rpc(self,
                     ],
                     mount_tokens=True,
                     resources=resources,
+                    attributes={'name': 'driver'},
```

Comment:
Take a look at line 350, there's some name logic already included. I think you should add your changes there.

Reply:
Do you want the job name to be $batch_name + 'driver'? 350 sets the batch attributes whereas here I'm just naming the driver job